### PR TITLE
D3ASIM-3073: Deprecated appliance.

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

File: src/d3a/models/base.py [hunk 1]
```diff
@@ -25,12 +25,10 @@
 
 
 class AreaBehaviorBase:
-    """Base class used by area behaviour defining classes (`BaseStrategy`, `BaseAppliance`)"""
+    """Base class used by area behaviour defining classes `BaseStrategy`"""
     def __init__(self):
-        # `area` is the area we trade in
```
Comment (spyrostz) [inline src/d3a/models/base.py:30]: I think that these comments are useful, please leave these here. Thanks!

File: src/d3a/models/state.py [hunk 2]
```diff
@@ -27,20 +27,6 @@
 
 StorageSettings = ConstSettings.StorageSettings
 
-# Complex device models should be split in three classes each:
```
Comment (spyrostz) [inline src/d3a/models/state.py:30]: This comment is important to keep here IMO. We already violate this a lot by not assigning enough things to the state.

**Review by spyrostz — APPROVED** [review]
Some small comments, LGTM once dealt with.

File: src/d3a/models/base.py (hunk 1 above)
Comment (hannesdiedrich) [reply src/d3a/models/base.py:30]: Agreed! I thought they are just wrong, but no, they are right!. Will re-add them

File: src/d3a/models/state.py (hunk 2 above)
Comment (hannesdiedrich) [reply src/d3a/models/state.py:30]: Ok, keeping in mind our offline discussion i will re-add and slightly change this comment.

**Review by spyrostz — DISMISSED** [review]
Still LGTM

**Pull request merged**